pdc/apps/auth/signals.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2018 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+
+
+def update_resources(sender, **kwargs):
```

Comment:
Could be skipped for tests by adding a new option to `settings_test.py` and checking the value here (deleting these objects in tests won't be needed).

This function can be also tested, simply by calling this function and checking that a known resource is available.